tests/CMakeLists.txt

```diff
@@ -205,6 +211,11 @@ if(QUDA_CONTRACT)
   target_link_libraries(contract_test ${TEST_LIBS})
   quda_checkbuildtest(contract_test QUDA_BUILD_ALL_TESTS)
   install(TARGETS contract_test ${QUDA_EXCLUDE_FROM_INSTALL} DESTINATION ${CMAKE_INSTALL_BINDIR})
+  
+  #add_executable(correlator_test correlator_test.cpp)
```

Comment:
Why is this commented out?

Reply:
We don't need it, deleted in d62ea10bddc4bdf8621ec711c5adf628f13913ff